File: auto-docs/executables/nodejs/legend_visibility.js
var plotly = require('plotly')('TestBot', 'r1neazxo9w')

{}

var graph_options = {filename: "legend-visibility", fileopt: "overwrite", layout: layout, auto_open: "false"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});

File: auto-docs/executables/julia/legend_visibility.jl
using Plotly
Plotly.signin("TestBot", "r1neazxo9w")

trace1 = [
  "x" => [0, 1, 2, 3, 4, 5, 6, 7, 8], 
  "y" => [0, 3, 6, 4, 5, 2, 3, 5, 4], 
  "type" => "scatter"
]
trace2 = [
  "x" => [0, 1, 2, 3, 4, 5, 6, 7, 8], 
  "y" => [0, 4, 7, 8, 3, 6, 3, 3, 4], 
  "type" => "scatter"
]
data = [trace1, trace2]
layout = ["showlegend" => false]
response = Plotly.plot(data, ["layout" => layout, "fileopt" => "overwrite", "filename" => "legend-visibility"])
plot_url = response["url"]